### ISSUE TITLE
Weird rendering of annotations

### ISSUE THREAD

**Issue by bobthemighty** [adamreisnz/seki#2]
Hello!

I have a few boards for this talk where the moves are adding annotations rather than playing a game.

For example, this board shows how white, having captured three stones at the end of a game, uses them to fill black's territory.

```
(;PB[Black]PW[White]SZ[9]AB[bf][cc][cd][ce][cg][ch][db][dd][dh][ea][eb][ec][ed][eh][ei][fa][fc][fh][id][ie][if]AW[cf][de][df][dg][ee][eg][fb][fd][fg][fi][ga][gb][gc][gd][gg][gh][gi][hi]PL[B]AP[Sabaki:0.52.2]CA[UTF-8]AE[dc][ci][di]LB[id:1][ie:2][if:3];PL[W]AB[dc]AE[id]LB[dc:1][ie:2][if:3];PL[B]AB[dc][di]AE[id][ie]LB[dc:1][di:2][if:3];PL[W]AB[dc][di][ci]AE[id][ie][if]LB[ci:3][dc:1][di:2])
```

In seki, it renders the final position, with all the annotations, whereas in most sgf viewers, the three stones appear at the right, and are moved one by one to their final positions: http://eidogo.com/#3PahfbF6J

I'm loving the lib btw, and apologise for finding bugs.

**Comment by adamreisnz** [comment]
Hi!

Don't worry about it, happy to look into these. As the library is quite new it hasn't been used extensively yet so bugs are probably a bit more likely that other libraries, so it's great to get the feedback and bug reports so I can address them.

This is interesting, I don't think I've encountered this kind of annotation before, but I will try to add support for it shortly!

**Comment by adamreisnz** [comment]
This should be fixed in `1.8.13`!

See [working example](https://letsplaygo.com/editor?sgf=(;PB[Black]PW[White]SZ[9]AB[bf][cc][cd][ce][cg][ch][db][dd][dh][ea][eb][ec][ed][eh][ei][fa][fc][fh][id][ie][if]AW[cf][de][df][dg][ee][eg][fb][fd][fg][fi][ga][gb][gc][gd][gg][gh][gi][hi]PL[B]AP[Sabaki:0.52.2]CA[UTF-8]AE[dc][ci][di]LB[id:1][ie:2][if:3];PL[W]AB[dc]AE[id]LB[dc:1][ie:2][if:3];PL[B]AB[dc][di]AE[id][ie]LB[dc:1][di:2][if:3];PL[W]AB[dc][di][ci]AE[id][ie][if]LB[ci:3][dc:1][di:2])) with your SGF.
